packages/protocol/contracts/L1/hooks/AssignmentHook.sol

```diff
@@ -33,6 +33,7 @@ contract AssignmentHook is EssentialContract, IHook {
         uint64 maxBlockId;
         uint64 maxProposedIn;
         bytes32 metaHash;
+        bytes32 parentMetaHash;
```

Comment:
@davidtaikocha We should deploy a new hook instead of upgrading the old one. Provers who don't care about the parentMataHash can keep using the previous hook, and some provers can opt in the new hook.

For clarity, when you deploy this contract again, you can rename the contract to 'AssignmentHook2' without making the same change in code.

Reply:
Sure, ok, will.